drivers/filesystems/nfs/nfs41_driver.c

```diff
@@ -4546,6 +4546,8 @@ NTSTATUS nfs41_QueryVolumeInformation(
     status = check_nfs41_dirquery_args(RxContext);
     if (status) goto out;
 
+    RtlZeroMemory(RxContext->Info.Buffer, RxContext->Info.LengthRemaining);
```

Comment:
Please, wrap this change with `#ifdef __REACTOS__` like it's done in other places here

Reply:
Thank you. I fixed.

Reply:
@khanhnt2: Do you think this line of code should also be added as PR in https://github.com/microsoft/Windows-driver-samples?

Reply:
@Saibamen I couldn't find nfs driver in Microsoft samples. Could you give me a link, I'm pleasure to do that.

Reply:
Sorry, this file is from University of Michigan, not Microsoft.